orte/mca/rmaps/base/rmaps_base_map_job.c

```diff
@@ -532,6 +462,7 @@ void orte_rmaps_base_map_job(int fd, short args, void *cbdata)
             }
             free(output);
         }
+        exit(1);
```

Comment:
Yeah, these are some leftovers. I created a "Work-In-Progress, Do-Not-Merge!" label and marked this with it. I'm trying to let some other folks poke at it to see if it does what they need, then they will augment and we need to cleanup prior to merge.

Still, I do appreciate the review! So please feel free to poke at it. I know there are several places that still require cleanup.